app/Models/Post.php

```diff
@@ -90,6 +90,12 @@ public function getTextAttribute($original)
 
         $text = str_replace('&lt;', '<', $text);
 
+        $text = str_replace(
+            ['<iframe', '</iframe>'],
```

Comment:
In the near future this function will be removed. I'd prefer a solution without having to replace html.

Reply:
To keep the same aspect ratio and make the iframe responsive, we need to wrap it with a `div`. So we wrap it programmatically or manually in the editor.